qemu/tests/cfg/cpu_device_hotplug.cfg

```diff
@@ -0,0 +1,80 @@
+- cpu_device_hotplug_test: install setup image_copy unattended_install.cdrom
+    mem = 4096
+    virt_test_type = qemu
+    type = cpu_device_hotplug
+    cpu_hotplug_timeout = 2000
+    n_cpus_add = 1
+    kill_vm = yes
+    iterations = 5
+    onoff_iterations = 100
+    login_timeout = 240
+    kill_vm = yes
+    only smp2
+    smp = 8
+    vcpu_maxcpus = 255
+    auto_cpu_model = "no"
+    cpu_hotplug_cmd = 'device_add CPU_MODEL-spapr-cpu-core,id=core{0},core-id={0}'
+    cpu_hotunplug_cmd = 'device_del core{0}'
+    variants:
+        - add_one_cpu:
+
+        - add_32_cpu:
+            n_cpus_add = 32
+        - add_64_cpu:
+            n_cpus_add = 64
+        - add_96_cpu:
+            n_cpus_add = 96
+        - add_128_cpu:
+            n_cpus_add = 128
+        - add_160_cpu:
+            n_cpus_add = 160
+        - add_192_cpu:
+            n_cpus_add = 192
+        - add_224_cpu:
+            n_cpus_add = 224
+        - add_max_cpu:
+            n_cpus_add = 255
+        - nonexist_vcpu:
+            type = nonexist_vcpu_hotplug
+            nonexist_vcpu = -1 256
+       - remove_one_cpu:
+            unplug = "yes"
+
+        - remove_32_cpu:
+            unplug = "yes"
+            n_cpus_remove = 32
+        - remove_64_cpu:
+            unplug = "yes"
+            n_cpus_remove = 64
+        - remove_96_cpu:
+            unplug = "yes"
+            n_cpus_remove = 96
+        - remove_128_cpu:
+            unplug = "yes"
+            n_cpus_remove = 128
+        - remove_160_cpu:
+            unplug = "yes"
+            n_cpus_remove = 160
+        - remove_192_cpu:
+            unplug = "yes"
+            n_cpus_remove = 192
+        - remove_224_cpu:
+            unplug = "yes"
+            n_cpus_remove = 224
+        - remove_max_cpu:
+            unplug = "yes"
+            n_cpus_remove = 255
+
+    variants:
+        - smp8_c1_t4:
+            smp = 8
+            used_cpus = 8
+            vcpu_cores = 1
+            vcpu_threads = 4
+            vcpu_maxcpus = 252
```

Comment:
qemu for x86/x86_64 only supportted max vcpus number  is 240, below is qemu output on my host:
[qemu output] Warning: Number of hotpluggable cpus requested (252) exceeds the recommended cpus supported by KVM (240)
[qemu output] Number of hotpluggable cpus requested (252) exceeds the maximum cpus supported by KVM (240)
QEMU emulator version 2.6.0

so,  I guess you need to add configuration like:
```
           vcpu_maxcpus = 240
           pserial:
                   vcpu_maxcpus = 252
```